# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

Header header
float32 sideLeft
float32 sideRight
float32 rearLeft
float32 rearCenter
float32 rearRight
